### PR TITLE
3.2.0 CLI Release notes

### PR DESCRIPTION
**Pull request by WillXuCodes**
- Created CLI Release Notes
- Updated mentions of prosv5 to pros where applicable.

### CODE REVIEW

**Review by kunwarsahni01 — APPROVED** [review]

**Review by HotelCalifornia — CHANGES_REQUESTED** [review]
did we release 3.1.4 or 3.2.0? because the release notes say 3.1.4.

also the windows installers are still broken (beyond repair?) so idk when we're gonna be able to actually get that out

File: v5/releases/cli3.2.0.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- Fix issue with schema length for upcoming vexOS release
+- Fix uploading without a project
+
+Updating to the new version
+===========================
+
+Python PIP
+----------
+
+Run ``pip install https://github.com/purduesigbots/pros-cli/releases/download/3.2.0/pros_cli_v5-3.2.0-py3-none-any.whl`` (or pip3, depending on your system)
```
Comment (HotelCalifornia) [inline v5/releases/cli3.2.0.rst:41]:
```suggestion
Run ``pip install --upgrade pros-cli` (or pip3, depending on your system)
```

**Review by HotelCalifornia — CHANGES_REQUESTED** [review]

File: v5/getting-started/linux.rst [hunk 2]
```diff
@@ -26,7 +26,7 @@ Installing the CLI
 
 1. If you do not already have one installed, install a version of Python greater than or equal to 3.6
 2. Check the latest version of the PROS CLI on `our releases page <https://github.com/purduesigbots/pros-cli3/releases/latest>`_, and run :code:`python3.6 -m pip install --user https://github.com/purduesigbots/pros-cli/releases/download/3.X.X/pros_cli_v5-3.X.X-py3-none-any.whl`, replacing the number after 'python' with the version you installed and the Xs with the numbers you found before. If you wish to install for all users, run the command with :code:`sudo` and remove the :code:`--user` flag.
```
Comment (HotelCalifornia) [inline v5/getting-started/linux.rst:28]:
```suggestion
2. Install the latest version of the PROS CLI from the Python Package Index, using :code:`python3 -m pip install --user pros-cli`. If you wish to install for all users, run the command with :code:`sudo` and remove the :code:`--user` flag.
```

File: v5/releases/cli3.2.0.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+We are also pleased to announce that pros-cli is back on PyPi! Anyone who installed through pip should be able to run pip install --upgrade pros-cli and get this release.
+
+A number of other miscellaneous bugfixes and improvements were added, so feel free to browse the full changelog below.
+
+New features:
+
+- Added wireless terminal
+- Rename distribution from pros-cli-v5 to pros-cli
+- Both `pros` and `prosv5` work for the CLI now.
+
```
Comment (HotelCalifornia) [inline v5/releases/cli3.2.0.rst:23]: add "compatible with vexOS 1.0.13"

File: v5/releases/cli3.2.0.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Windows
+-------
+
+Download and run the appropriate (.exe) installer from `our release page <https://github.com/purduesigbots/pros-cli/releases/3.2.0>`_.
+
+macOS
+-----
+
+To upgrade a previous installation, run ``brew upgrade pros-cli``.
+If you have the editor installed, you can also run ``brew cask upgrade pros-editor``.
```
Comment (HotelCalifornia) [inline v5/releases/cli3.2.0.rst:51]:
```suggestion
If you have the editor installed, you can also run ``brew upgrade pros-editor``.
```
i think the unification means you can just do this now

File: v5/releases/cli3.2.0.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+To install from scratch:
+
+Install Homebrew if it's not already installed. This may take a while.
+Run ``brew tap osx-cross/arm && brew tap purduesigbots/pros``
+Run ``brew install gcc-arm-bin && brew cask install pros-editor`` (if you want to install only the CLI, you can run ``brew install gcc-arm-bin pros-cli`` instead)
```
Comment (HotelCalifornia) [inline v5/releases/cli3.2.0.rst:57]:
```suggestion
Run ``brew install gcc-arm-bin && brew --cask install pros-editor`` (if you want to install only the CLI, you can run ``brew install gcc-arm-bin pros-cli`` instead)
```

File: v5/releases/cli3.2.0.rst [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+Notes:
+~~~~~~
+
+- if Homebrew complains ("It seems there is already an App at '/usr/local/Caskroom/pros-editor/1.32.2/PROS Editor.app'") when trying to upgrade the editor, run ``brew cask upgrade --force pros-editor`` instead.
+- if you already have the PROS CLI installed, or if Homebrew complains about some files existing when trying to install the CLI, you'll want to uninstall it first
+- if you installed using pip, you can run pip3 uninstall pros-cli-v5
+- if you installed using the .app bundle, you can move that bundle to the trash folder, and then run ``rm -f /usr/local/bin/prosv5`` to clear out the old file
```
[the remark this answers is not in the document]
Comment (HotelCalifornia) [reply v5/releases/cli3.2.0.rst:65]: these were more pertinent when i could reasonably expect people to have old, non-homebrew installs. by-and-large this is no longer the case

**Review by HotelCalifornia — CHANGES_REQUESTED** [review]

File: v5/releases/cli3.2.0.rst [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+-----
+
+To upgrade a previous installation, run ``brew upgrade pros-cli``.
+If you have the editor installed, you can also run ``brew upgrade pros-editor``.
+
+To install from scratch:
+
+Install Homebrew if it's not already installed. This may take a while.
+Run ``brew tap osx-cross/arm && brew tap purduesigbots/pros``
+Run ``brew install gcc-arm-bin && brew --cask install pros-editor`` (if you want to install only the CLI, you can run ``brew install gcc-arm-bin pros-cli`` instead)
```
Comment (HotelCalifornia) [inline v5/releases/cli3.2.0.rst:57]:
```suggestion
If you are installing for the first time, see `the getting started guide for macOS <https://pros.cs.purdue.edu/v5/getting-started/macos.html>` for instructions.
```

**Review by HotelCalifornia — APPROVED** [review]

**Pull request merged**